src/hooks/useAccounts.ts

```diff
@@ -145,7 +145,7 @@ export const useAccountConnect = () => {
         walletName: combinedWallet.evm.name,
         chainType: ChainType.EVM,
         chainId: await combinedWallet.evm.getChainId(),
-        address: `${await combinedWallet.evm.getAccounts()}`,
+        address: `${(await combinedWallet.evm.getAccounts())[0]}`,
```

Comment:
Nice! We might also adjust that part for the user tracking in regards of arcx. They stated that multiple wallets are sent.